project/docs/macros.py

```diff
@@ -31,9 +31,9 @@ def get_credits_data() -> dict:
     dependencies = direct_dependencies | indirect_dependencies
 
     packages = {}
-    for pkg in search_packages_info(dependencies):
-        pkg = {_: pkg[_] for _ in ("name", "home-page")}
-        packages[pkg["name"].lower()] = pkg
+    for pkg in search_packages_info(list(dependencies)):
```

Comment:
this one looked like easy cast to list so that is what I did here. Assuming should be fine unless it was supposed to be that way before.